Licenses/COPYING_7ZIP.txt

```diff
@@ -0,0 +1,7 @@
+LZMA SDK is written and placed in the public domain by Igor Pavlov.
+
+You can copy, modify, distribute and perform LZMA SDK code, even for
```

Comment:
Once merged we need to pull in the licenses into the licensing folder.

Reply:
Yes, we need to